src/main/java/duke/services/UI.java

```diff
@@ -0,0 +1,34 @@
+package duke.services;
+
+/** Handles display */
+public class UI {
```

Comment:
Perhaps the name of this class would be better written as Ui, to conform to the coding standard?